modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js

```diff
@@ -299,7 +297,11 @@ return view.extend({
 				return true;
 			};
 
+			o.port_opts = [];
+			var port_opts = o.port_opts;
```

Comment:
@ptpt52 - this line (301) seems redundant. Can you please confirm that your fix still works after removing it?

Reply:
no, it is not redundant.

Reply:
The the code should be refactored. Having a `var port_opts` in both the outer and inner scope is confusing.

Reply:
https://github.com/openwrt/luci/blob/c7cae787e18a4eea647978bb7ac6dae5fbad7d8b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js#L367
https://github.com/openwrt/luci/blob/c7cae787e18a4eea647978bb7ac6dae5fbad7d8b/modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js#L370

Reply:
If convenient you can help rename it

Reply:
Yep seen it now, could you please change it like that:
Merge line 300 & 301 into a chained assignment
```javascript
var port_opts = o.port_opts = [];
```